test.py

```diff
@@ -168,10 +168,14 @@ def check_or_fail(condition,
 
 
 def check_version_found(plugin_version):
-    # This function should check that version in <p> element with class
-    # 'version' is equal to plugin version you provided with variable.
-    # You may check out one of other functions as example.
-    return False
+    try:
+        if plugin_version in driver.find_element_by_css_selector(
+                "p[class='version']").text:
```

Comment:
You found out the text, but where do you compare with provided plugin version?

Reply:
I don't get it. The ```plugin_version``` value is fetched from the ENV variable passed when running the container. Tests were passing when ```v0.10.0``` was indicated, and test were failing when another version was indicated (i.e. ```v0.11.0```)

That's not the expected behavior?

Reply:
you are correct. I overlooked that you checked `plugin_version`.